Tests/conf.json

```diff
@@ -5374,14 +5374,14 @@
         "Phishing v3 - DomainSquatting+EML+MaliciousIndicators - Test": "Issue CRTX-70684"
     },
     "skipped_integrations": {
-        "Lastline v2": "No license",
-        "AbuseIPDB": "License expired",
-        "checkpointdome9": "No license",
+        "Lastline v2": "No instance - No license",
+        "AbuseIPDB": "No instance - License expired",
+        "checkpointdome9": "No instance - No license",
         "_comment1": "~~~ NO INSTANCE ~~~",
```

Comment:
make sure the validation is not case sensitive 💯

Reply:
it is not :)